var/spack/repos/builtin/packages/llvm-doe/package.py

```diff
@@ -0,0 +1,525 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os.path
+import re
+import sys
+
+import llnl.util.tty as tty
+import spack.util.executable
+
+
+class LlvmDoe(CMakePackage, CudaPackage):
+    """This package provides a collection of the experimental LLVM projects done
+    by the US DOE research and development teams.
+    """
+
+    homepage = "https://github.com/llvm-doe-org"
+    url = "https://github.com/llvm-doe-org/llvm-project/archive/llvmorg-10.0.0.zip"
+    git = "https://github.com/llvm-doe-org/llvm-project"
+    maintainers = ['shintaro-iwasaki']
+
+    family = "compiler"  # Used by lmod
```

Comment:
Where is this attribute used by `lmod`?

Reply:
Thanks. `family = ` seems unnecessary as you pointed out.  I removed it.